src/js/common/schemaform/uiDefinitions.js

```diff
@@ -0,0 +1,37 @@
+import { validateSSN, validateDate } from './validation';
+
+export const uiFullName = {
+  first: {
+    'ui:title': 'First name'
```

Comment:
To make it easier to share the schema with the backend, yes. I think the more stuff we put in uiSchema the better, since we can update it directly in this repo and don't have to worry about shared backend code.